src/components/SocialBar/SocialBar.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { socialBarTestkitFactory } from '../../testkit/protractor';
+
+/**
+ * For tests containing interactions.
+ * Can be removed if not used.
+ * */
+describe('socialBar', () => {
+  const storyUrl = createStoryUrl({
+    kind: 'Components/Share',
+    story: 'SocialBar',
+    withExamples: true,
+  });
+  const dataHook = 'storybook-SocialBar';
+
+  beforeEach(() => browser.get(storyUrl));
+
+  it('should render', async () => {
+    const driver = socialBarTestkitFactory({ dataHook });
+    await waitForVisibilityOf(await driver.element(), 'Cannot find SocialBar');
+    expect((await driver.element()).isDisplayed()).toBe(true);
+  });
+});
```

Comment:
I think this file is not needed.
The enzyme test file is enough

Reply:
deleted